src/signals/incident-management/containers/Dashboard/RealtimeDashboard.tsx

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import { StyledMenuInline, MenuChildren } from '@amsterdam/asc-ui'
+const RealtimeDashboard = () => {
+  return (
+    <>
+<StyledMenuInline>
```

Comment:
indention

Reply:
I had to put the setting back that you changed in Pycharm but it took a while for me to notice that was necessary.